redux/slices/theme.ts

```diff
@@ -2,7 +2,7 @@ import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 
 export type Theme = "light" | "dark";
 
-export const defaultTheme = "dark";
+export const defaultTheme = "light";
```

Comment:
?

Reply:
We have the different default themes in MUI and in style of our project. You add the `.dark-theme` class after the check of localStorage. That it why you have the blinking in light theme in components from MUI